.trunk/trunk.yaml

```diff
@@ -20,7 +21,7 @@ lint:
   ignore:
     - linters: [ALL]
       paths:
-        - "**/test_data" # required for golangci-lint, which runs on directories
```

Comment:
nit: I think this is useful to leave in on account of it looking odd that we require two seemingly identical globs

Reply:
ya thing is I thought this was actually for multiple linters and was more nuanced than the comment implies. Also we need to eliminate the need for this

Reply:
I see. IIRC it's just for golangci-lint at this point, since it's the only one with directory targets that we have turned on in this repo. It's not the easiest to eliminate the need for this, but I agree we should

Reply:
fair enough - will re-add